File: 4. (27-03) Lista de Exercícios #1/Beecrowd_1041.py
'''
Beecrowd: Problema 1041
25/03/2024
Sophia Swiercoswski
'''

x, y = map(float, input(). split())

if x > 0 and y > 0:
    print ("Q1")
elif x < 0 and y > 0:
    print ("Q2")
elif x < 0 and y < 0:
    print ("Q3")
elif x > 0 and y < 0:
    print ("Q4")
elif x == 0 and y != 0:
    print ("Eixo Y")
elif x != 0 and y == 0:
    print ("Eixo X")
else:
    print ("Origem")

File: Algoritmos e programação I/11. (21-08) Lista de Exercícios #4/Beecrowd_1041.cpp
/*
Beecrowd: Problema 1041
08/08/2024
Sophia Swiercoswski
*/

#include <iostream>
using namespace std;

int main() {
    // Ler dois números decimais do usuário
    float x, y;
    cin >> x >> y;

    // Determinar qual quadrante os números pertencem
    if(x > 0 && y > 0) {
        printf("Q1\n");
    }
    else if(x < 0 && y > 0) {
        printf("Q2\n");
    }
    else if(x < 0 && y < 0) {
        printf("Q3\n");
    }
    else if(x > 0 && y < 0) {
        printf("Q4\n");
    }
    else if(y == 0 && x != 0) {
        printf("Eixo X\n");
    }
    else if(x == 0 && y != 0) {
        printf("Eixo Y\n");
    }
    else {
        printf("Origem\n");
    }

    return 0;
}